graphistry/util.py

```diff
@@ -21,7 +21,10 @@ def setup_logger(name, verbose=VERBOSE, fullpath=TRACE):
         FORMAT = " %(message)s\n"
     logging.basicConfig(format=FORMAT)
     logger = logging.getLogger(f'graphistry.{name}')
-    logger.setLevel(logging.INFO if verbose else logging.DEBUG)
+    if verbose is None:
```

Comment:
we need to kill `verbose` anti-patterns here, i'll file an issue to get this stuff normal